packages/components/forms/src/base-input/types.ts

```diff
@@ -15,9 +15,9 @@ export interface BaseInputInternalProps extends CommonProps {
    */
   as?: 'input' | 'textarea'; // TODO: we can extend to select
   /**
-   * Label value is set as aria-label attribute in the input
+   * Aria-label attribute in the input
    */
-  label: string;
+  ariaLabel: string;
```

Comment:
I think it should be optional, right?

Reply:
hmm, I think we need to enforce it. If not we can have not accessible input used by the client (someone can decide that they don't need the label for some reason)?

Reply:
I haven't seen it in other libraries. Do we still need it if an input is wrapped in a `label` tag?

Reply:
No, in that case, we wouldn’t need it. but what if user fogot about adding the label for some reason? I just thought we would have aria-label always, no matter if there is a label or not. 🤷‍♀️